nengo_gui/components/netgraph.py

```diff
@@ -487,6 +487,8 @@ def expand_network(self, network, client):
         for ens in network.ensembles:
             self.create_object(client, ens, type='ens', parent=parent)
         for node in network.nodes:
+            if node.output is None:
+                node.output = Slider.passthrough_fcn
```

Comment:
Do we need this to happen in the NetGraph?  I'd only change the ones that there are Sliders for, and Slider's already taking care of this.